corehq/form_processor/backends/sql/dbaccessors.py

```diff
@@ -380,6 +380,18 @@ def iter_forms_by_last_modified(start_datetime, end_datetime):
         )
 
     @staticmethod
+    def iter_form_ids_by_xmlns(domain, xmlns=None):
+        from corehq.sql_db.util import run_query_across_partitioned_databases
+
+        q_expr = Q(domain=domain)
```

Comment:
Should also filter by `state = XFormInstanceSQL.NORMAL`

Reply:
https://github.com/dimagi/commcare-hq/pull/18460/commits/4a69360c141b703cf2883ebecd860e69703dd2b3